libra-swarm/src/swarm.rs

```diff
@@ -242,11 +242,17 @@ impl LibraSwarm {
         for i in 0..num_launch_attempts {
             info!("Launch swarm attempt: {} of {}", i, num_launch_attempts);
 
+            let template = if let Some(template_path) = template_path.clone() {
+                NodeConfig::load(workspace_builder::workspace_root().join(template_path))
+                    .unwrap_or_default()
+            } else {
+                NodeConfig::default()
+            };
```

Comment:
I thought we were gonna change the interface here to take in the node config and not the file?